### ISSUE TITLE
Dictionary and Vector Support

### ISSUE THREAD

**Issue by vovavovavovavova** [StdCarrot/Py3AMF#17]
It seems, the Python Py3amf does not have last 2 objects supported from this list:

```
undefined-marker=0x00 
null-marker=0x01 
false-marker=0x02 
true-marker=0x03 
integer-marker=0x04
double-marker=0x05 
string-marker=0x06 
xml-doc-marker=0x07 
date-marker=0x08 
array-marker=0x09 
object-marker=0x0A 
xml-marker=0x0B 
byte-array-marker=0x0C
vector-int-marker=0x0D
vector-uint-marker=0x0E
vector-double-marker=0x0F
vector-object-marker=0x10
dictionary-marker=0x11
```

If I am wrong, please tell me where. When I try to parse amf3 content that has dictionary, it fails.

**Comment by hadamrd** [comment]
I added a pull request that solves the problem.
I have also the fix implemented in my fork :
https://github.com/hadamrd/Py3AMF

**Comment by StdCarrot** [comment]
it is solved with 0.8.12
